src/runtime/tests/capabilities-resolver-test.ts

```diff
@@ -20,50 +23,83 @@ import {Runtime} from '../runtime.js';
 import {MockFirebaseStorageDriverProvider} from '../storageNG/testing/mock-firebase.js';
 
 describe('Capabilities Resolver', () => {
-  it('creates storage keys', () => {
+  type StorageKeyType = typeof VolatileStorageKey|typeof RamDiskStorageKey|typeof DatabaseStorageKey;
+  function verifyStorageKey(key: StorageKey, expectedType: StorageKeyType) {
+    assert.isTrue(key instanceof ReferenceModeStorageKey);
+    const refKey = key as ReferenceModeStorageKey;
+    assert.instanceOf(refKey.backingKey, expectedType);
+    assert.instanceOf(refKey.storageKey, expectedType);
+  }
+  async function verifyThrows(promise: Promise<StorageKey>) {
+    try {
+      await promise;
+      assert.fail();
+    } catch (e) {
+      // do nothing.
+    }
+  }
```

Comment:
You can just use `assertThrowsAsync` from `src/testing/test-util.ts` instead.

Reply:
oh, thank you! i knew there should be something like that, but couldn't find it...